Makefile

```
@@ -1,26 +1,14 @@
+#### import version information report
+include VersionReport
+###
```

```suggestion
include VersionReport

```

Nit: this comment is redundant; the code is self-describing.